content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -73,6 +73,10 @@ kubectl config set-context gce --user=cluster-admin --namespace=foo \
   && kubectl config use-context gce
 
 kubectl config unset users.foo                       # delete user foo
+
+# short alias to set/show context/namespace (useful for CKAD/CKA exams)
```

Comment:
It's be better to call out that this only works in `bash` and compatible shells. You also have to have a current context for the error handling to work here:
```
$ kn no-such-namespace
error: no current context is set
error: current-context must exist in order to minify
$ 
```
---

Despite the term “cheat sheet” in the title, the page is mainly aimed at Kubernetes end users and exam candidates.
```suggestion
# short alias to set/show context/namespace
```

@mtilson I don't _object_ to this PR merging, but I'm also not inclined to approve it. Keeping the cheat sheet page concise is useful for our end user readers who aren't sitting exams.

Reply:
No problem, the main idea was to have, probably, the shortest way to setup something  (aliases in this case) in `shell` to juggle with contexts and namespaces. It is especially useful during exams, as there is no time to setup heavier plugins like `kubectx` or `kubens`, but it also could be useful for general end user who looks through this cheatsheet. And yes, as `bash` is the shell used during exam I tried to use its constructions, but it seems it is possible to convert them to POSIX compatible ones.